src/components/panel/panel.js

```diff
@@ -489,6 +482,16 @@ angular
 
 /**
  * @ngdoc method
+ * @name MdPanelPosition#tryPanelPositions
```

Comment:
Addressed for now by using just the one function `withPanelPosition`, which can be called multiple times.